src/Calculator.ts

```diff
@@ -248,3 +250,56 @@ const POPULATION_DISPLAY_MAP = {
   [PopulationType.MSRPOPLEX]: 'Measure Population Exclusion',
   [PopulationType.OBSERV]: 'Measure Observation'
 };
+
+function addSDE(report: R4.IMeasureReport, measureURL: string, result: ExecutionResult) {
+  // 	Note that supplemental data are reported as observations for each patient and included in the evaluatedResources bundle. See the MeasureReport resource or the Quality Reporting topic for more information.
+  result.supplementalData?.forEach(sd => {
+    const observation = <R4.IObservation>{};
+    observation.resourceType = 'Observation';
+    observation.code = { text: sd.name };
+    observation.id = uuidv4();
+    observation.status = R4.ObservationStatusKind._final;
+    observation.extension = [
+      {
+        url: 'http://hl7.org/fhir/StructureDefinition/cqf-measureInfo',
+        extension: [
+          {
+            url: 'measure',
+            valueCanonical: measureURL
+          },
+          {
+            url: 'populationId',
+            valueString: sd.name
+          }
+        ]
+      }
+    ];
+
+    // add coding to valueCodeableConcept
+    if ('forEach' in sd.rawResult) {
+      observation.valueCodeableConcept = { coding: [] };
+      sd.rawResult?.forEach((rr: any) => {
+        if (rr.code && rr.system && rr.display && rr.code.value && rr.system.value && rr.display.value) {
+          // create supplemental data elements
+          observation.valueCodeableConcept?.coding?.push({
+            system: rr.system.value,
+            code: rr.code.value,
+            display: rr.display.value
+          });
+        }
+      });
+    } else if (sd.rawResult.isCode) {
```

Comment:
Does this property actually come from anywhere? Is this something that we anticipate being in an SDE in the future, or are we going to set it ourselves? I couldn't find it anywhere.

Reply:
Hoss suggested this, and it does come up for sde-sex in the EXM130 example. Basically, sometimes the rawResult is directly a code (rather than our other cases, which is a FHIR object), which responds to the isCode property.

Reply:
https://github.com/cqframework/cql-execution/blob/decaffeinate-2020/src/datatypes/clinical.js#L11